### PR TITLE
Get rid of `expect_identical_linter()` lints

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: tests/testthat/test-methods.R [hunk 1]
```diff
@@ -71,10 +71,7 @@ test_that("as.data.frame.lints", {
     stringsAsFactors = FALSE
   )
 
-  expect_equal(
-    df,
-    exp
-  )
+  expect_equal(df, exp)
```
Comment (MichaelChirico) [inline tests/testthat/test-methods.R:74]: what's going on here? 🙃

Comment (IndrajeetPatil) [reply tests/testthat/test-methods.R:74]: `df` columns are not of integer type here:

```r
── Failure (test-methods.R:74): as.data.frame.lints ────────────────────────────
`df` (`actual`) not identical to `exp` (`expected`).

`actual$line_number` is a double vector (1, 2)
`expected$line_number` is an integer vector (1, 2)

`actual$column_number` is a double vector (1, 6)
`expected$column_number` is an integer vector (1, 6)
```

Comment (IndrajeetPatil) [reply tests/testthat/test-methods.R]: Was going to fix it in a different PR, but might as well do it here.

Comment (MichaelChirico) [reply tests/testthat/test-methods.R]: thanks, I would have suggested just leaving the style fix for that other PR

Comment (IndrajeetPatil) [reply tests/testthat/test-methods.R]: So should I undo this here?

Comment (MichaelChirico) [reply tests/testthat/test-methods.R]: no it's fine, minor point here

**Review by MichaelChirico — APPROVED** [review]

**Comment by MichaelChirico** [comment]
madman, you actually did it! 🤟🤟

**Pull request merged**